src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from "@angular/core";
+import { GlobalDataService } from "src/app/globalData.service";
+import { IGlobal } from "src/app/global";
+import { GoogleChartInterface } from "ng2-google-charts";
+
+@Component({
+  selector: "app-home",
+  templateUrl: "./home.component.html",
+  styleUrls: ["./home.component.scss"],
+})
+export class HomeComponent implements OnInit {
+  NewConfirmed = 0;
+  TotalConfirmed = 0;
+  TotalDeaths = 0;
+  TotalRecovered = 0;
+  Date = "";
+  Country = "";
+  data: IGlobal[];
+  pieChart: GoogleChartInterface = {
+    chartType: "PieChart",
+  };
+  lineChart: GoogleChartInterface = {
+    chartType: "LineChart",
+  };
+  isShowDivIf = true;
+
+  toggleDisplayDivIf() {
+    this.isShowDivIf = !this.isShowDivIf;
+  }
+  constructor(private dataService: GlobalDataService) {}
+
+  initChart() {
+    let datatable = [];
+    datatable.push(["Country", "Cases"]);
+    this.data.forEach((el) => {
+      if (el.TotalConfirmed > 2000)
+        datatable.push([el.Country, el.TotalConfirmed]);
+    });
+    console.log(datatable);
+
+    this.pieChart = {
+      chartType: "PieChart",
+      dataTable: datatable,
+      options: { height: 450, pieHole: 0.4 },
+    };
+
+    this.lineChart = {
+      chartType: "LineChart",
+      dataTable: datatable,
+      options: { height: 450 },
+    };
+  }
+
+  ngOnInit() {
+    this.dataService.fetchGlobalData().subscribe({
+      next: (result) => {
+        console.log(result);
+        this.data = result;
+        console.log(this.data);
+        result.forEach((el) => {
+          if (!Number.isNaN(el.NewConfirmed)) {
+            this.NewConfirmed += el.NewConfirmed;
+            this.TotalConfirmed += el.TotalConfirmed;
+            this.TotalDeaths += el.TotalDeaths;
+            this.TotalRecovered += el.TotalRecovered;
```

Comment:
First of, if we are getting this data in an object, why aren't we using it like so, as well? 🤔 

Second, is this right (`+=`) ? Should we add those numbers to existing or should we use new numbers each time? (Actual question, maybe this is correct) 😸

Reply:
Actually I wont everything to calculate  from object property "Country" array (because i didn't noticed global data on start of object 🤔) and actually this is correct calculation if I catch data from county object array. I resolve this problem and now I fetch global data and data from country. For cards I use global data and for charts and table I use country data.